Tests/TuistKitTests/Services/InitServiceTests.swift

```diff
@@ -60,8 +60,11 @@ final class InitServiceTests: TuistUnitTestCase {
         templatesDirectoryLocator.templateDirectoriesStub = { _ in
             [defaultTemplatePath]
         }
-        let expectedAttributes = ["name": "Name", "platform": "macOS"]
-        var generatorAttributes: [String: String] = [:]
+        let expectedAttributes: [String: TuistGraph.Template.Attribute.Value] = [
+            "name": .string("Name"),
```

Comment:
I would use literals wherever possible

Reply:
Didn't notice it was TuistGraph, then it was fine to use enums. But it's also fine adding the expressibleBy IMO